File: ignored-test/java-junit/ConditionEvaluatorTests_ExceptionTest14.java
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Disabled;
import static org.junit.jupiter.api.Assertions.assertEquals;

import java.io.*;

public class ConditionEvaluatorTests {

    @Test
    @Disabled("")
    public void exceptionTest14() throws Exception {
        InvalidOperationException inner = new InvalidOperationException("f");
        ConditionParseException ex1 = new ConditionParseException("msg", inner);
        ByteArrayOutputStream bos = new ByteArrayOutputStream();
        ObjectOutputStream oos = new ObjectOutputStream(bos);
        oos.writeObject(ex1);
        oos.flush();
        ByteArrayInputStream bis = new ByteArrayInputStream(bos.toByteArray());
        ObjectInputStream ois = new ObjectInputStream(bis);
        Exception ex2 = (Exception) ois.readObject();
        assertEquals("msg", ex2.getMessage());
        assertEquals("f", ex2.getCause().getMessage());
    }
}

File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/ConditionEvaluatorTests_ExceptionTest14.cs
using Xunit;

namespace None
{
    public class ConditionEvaluatorTests
    {
        [Fact(Skip = "")]
        public void ExceptionTest14()
        {
            {
                var inner = new InvalidOperationException("f");
                var ex1 = new ConditionParseException("msg", inner);
                BinaryFormatter bf = new BinaryFormatter();
                MemoryStream ms = new MemoryStream();
                bf.Serialize(ms, ex1);
                ms.Position = 0;
                Exception ex2 = (Exception)bf.Deserialize(ms);
                Assert.Equal("msg", ex2.Message);
                Assert.Equal("f", ex2.InnerException.Message);
            }
        }
    }
}
